### PR TITLE
refactor(ci): shard e2e tests

### PR DESCRIPTION
**Pull request by petrjasek**

### CODE REVIEW

**Comment by petrjasek** [comment]
testing if travis can use sharding at all

File: server/app.py [hunk 1]
```diff
@@ -68,8 +79,6 @@ def get_app(config=None):
         json_encoder=MongoJSONEncoder,
         validator=SuperdeskValidator)
 
-    superdesk.app = app
```
Comment (ioanpocol) [inline server/app.py]: The superdesk.app package variable is used a few(34) times in code. Probably we have to change the code and use the current application instead.

Comment (petrjasek) [inline server/app.py]: yep right

File: client/protractor-conf.js [hunk 2]
```diff
@@ -21,6 +23,7 @@ exports.config = {
     },
 
     specs: ['spec/**/*[Ss]pec.js'],
+    //exclude: ['spec/authoring*.js'],
```
Comment (plamut) [inline client/protractor-conf.js]: Should be removed?

File: client/spec/helpers/auth.js [hunk 3]
```diff
@@ -25,9 +25,9 @@ function getToken(callback) {
             }
             if (!json.token) {
                 console.log(json);
-                throw new Error('Auth failed');
+                //throw new Error('Auth failed');
```
Comment (plamut) [inline client/spec/helpers/auth.js]: Remove completely?

File: server/scripts/start_e2e_server.sh [hunk 4]
```diff
@@ -7,6 +7,6 @@ ELASTICSEARCH_INDEX=superdesk_e2e
 REDIS_URL=redis://localhost:6379/2
 EOF
 
-honcho -e /tmp/testenv run python manage.py users:create -u admin -p admin -e admin@localhost --admin=true
-honcho -e /tmp/testenv start
+#honcho -e /tmp/testenv run python manage.py users:create -u admin -p admin -e admin@localhost --admin=true
```
Comment (plamut) [inline server/scripts/start_e2e_server.sh]: Remove comment?

**Comment by petrjasek** [comment]
will create a new PRs for both core/superdesk later

**Pull request closed without merging**